jobs/examples/ex1_frameworked_job.py

```diff
@@ -1,20 +1,24 @@
 """Same as ex1_full_sql_job.sql but allows access to spark for more complex ops (not used here but in ex2_frameworked_job.py)."""
 from yaetos.etl_utils import ETL_Base, Commandliner
+from pyspark import sql
 
 
 class Job(ETL_Base):
-    def transform(self, some_events, other_events):
-        df = self.query("""
+    def transform(
+        self, some_events="some_events", other_events="other_events"
```

Comment:
The inputs have to be spark dataframes or pandas dataframes. When they are spark dataframes, the framework register all of them into the spark SQL environment so they can be used directly inside queries, like those in `self.query()`.